### PR TITLE
Tidy garbage collection controller page

### PR DESCRIPTION
**Pull request by sftim**
Improve https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/

- fix YAML marked as shell
- tweak working

/priority backlog

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/garbage-collection.md [hunk 1]
```diff
@@ -45,19 +45,19 @@ kubectl get pods --output=yaml
 
 The output shows that the Pod owner is a ReplicaSet named `my-repset`:
 
-```shell
+```yaml
 apiVersion: v1
 kind: Pod
 metadata:
-  ...
+  …
```
Comment (kbhawkey) [inline content/en/docs/concepts/workloads/controllers/garbage-collection.md]: hello @sftim. I am not sure if the change on line 52 is needed. Have a look at the preview.

Comment (sftim) [reply content/en/docs/concepts/workloads/controllers/garbage-collection.md]: I think I could leave that change out; for me, it's an improvement to the page.

Comment (jaypipes) [reply content/en/docs/concepts/workloads/controllers/garbage-collection.md]: I agree with Karen that the long form is actually easier to read and would prefer reverting this particular change. At least, it's easier to read for my old eyes! :)

Comment (sftim) [reply content/en/docs/concepts/workloads/controllers/garbage-collection.md]: Tweaked

**Pull request merged**